edge/pkg/edgehub/process.go

```diff
@@ -53,26 +68,34 @@ func (eh *EdgeHub) dispatch(message model.Message) error {
 		md = modules.MetaGroup
 	case messagepkg.UserGroupName:
 		md = modules.BusGroup
-	default:
-		klog.Warningf("msg_group not found")
-		return fmt.Errorf("msg_group not found")
 	}
 
-	isResponse := eh.isSyncResponse(message.GetParentID())
+	isResponse := isSyncResponse(message.GetParentID())
 	if isResponse {
-		beehiveContext.SendResp(message)
+		beehiveContext.SendResp(*message)
 		return nil
 	}
 	if group == messagepkg.UserGroupName && message.GetSource() == "router_eventbus" {
-		beehiveContext.Send(modules.EventBusModuleName, message)
+		beehiveContext.Send(modules.EventBusModuleName, *message)
 	} else if group == messagepkg.UserGroupName && message.GetSource() == "router_servicebus" {
-		beehiveContext.Send(modules.ServiceBusModuleName, message)
+		beehiveContext.Send(modules.ServiceBusModuleName, *message)
 	} else {
-		beehiveContext.SendToGroup(md, message)
+		beehiveContext.SendToGroup(md, *message)
 	}
 	return nil
 }
 
+func (eh *EdgeHub) dispatch(message model.Message) error {
+	// handler for msg.
+	isProcessed, err := msghandler.ProcessHandler(message, eh.chClient)
+	if isProcessed {
+		return err
+	}
+
+	klog.Errorf("msg not processed due to not-supported, group: %v", message.GetGroup())
+	return fmt.Errorf("msg not processed due to not-supported, group: %v", message.GetGroup())
```

Comment:
Will always return error?

Reply:
all supported messages will be processed by processors. So if the codes run here, it means that it's not supported, so just return error